packages/experimental/TabList/src/Tab/useTab.win32.ts

```diff
@@ -74,19 +84,29 @@ export const useTab = (props: TabProps): TabInfo => {
     addTabKey(tabKey);
     // Set a defaultTabbableElement if we're the initial selectedKey.
     if (selectedKey === tabKey) {
-      componentRef && setSelectedTabRef(componentRef);
+      componentRef && setFocusedTabRef(componentRef);
     }
     return () => removeTabKey(tabKey);
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  React.useEffect(() => {
+    updateTabRef(tabKey, componentRef);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Please add a comment explaining why you're disabling the eslint warning

Reply:
Left a comment that amounted to saying that "the warning is disabled because dependencies are intentionally left out to prevent the hook from running when it shouldn't". 

Should I just take out the warnings?

Reply:
I think if it's intentional that the hook doesn't run when the function changes, this is fine.

The comment is so that people understand why the warning is silenced so that if someone comes through to fix warnings, they don't remove warning suppressions that are intentional. Also, it prevents us from being lazy and adding in warnings when we shouldn't.